### PR TITLE
merge latest Release v2.1.0

### PR DESCRIPTION
**Pull request by PhotoNomad0**
#### Describe what your pull request addresses (Please include relevant issue numbers):
-

#### Please include detailed Test instructions for your pull request:
-

#### Standard Test Instructions for PR Review Process:

- [ ] Double check unit tests that have been written
- [ ] Check for documentation for code changes
- [ ] Check that there are not inadvertent commits to tC Apps when reviewing a tC Core PR
- [ ] Checkout the branch locally and ensure that app runs as expected
  - [ ] Ensure tests pass
  - [ ] Open and watch the console for errors
  - [ ] Make sure all actions perform as expected
  - [ ] Import and Load a new Project
  - [ ] Load a tool and perform basic actions
  - [ ] Switch tools and perform basic actions
  - [ ] Switch project to an existing project
  - [ ] Load a tool and perform basic actions
  - [ ] Switch tools and perform basic actions
  - [ ] Next time reverse the order of importing after loading an existing project
- [ ] Reviewer should double check the DoD in the ISSUE, including the “spirit” of the story
- [ ] Ask Ben or Koz if you have any concerns about implementation (especially UI related)

---
This change is [Reviewable](https://reviewable.io/reviews/unfoldingword/translationcore/6537)

### CODE REVIEW

**Comment by dsm-git** [comment]
[Linux build 2.1.0 (d8f36fe)](https://tc-builds.door43.org/build/release-v2.1.0/translationCore-linux-x64-2.1.0-d8f36fe.deb) is ready for download.
[macOS build 2.1.0 (d8f36fe)](https://tc-builds.door43.org/build/release-v2.1.0/translationCore-macos-x64-2.1.0-d8f36fe.dmg) is ready for download.
[Windows x64 build 2.1.0 (d8f36fe)](https://tc-builds.door43.org/build/release-v2.1.0/translationCore-win-x64-2.1.0-d8f36fe.setup.exe) is ready for download.
[Windows x32 build 2.1.0 (d8f36fe)](https://tc-builds.door43.org/build/release-v2.1.0/translationCore-win-x32-2.1.0-d8f36fe.setup.exe) is ready for download.

File: src/locale/nonTranslatable.js [hunk 1]
```diff
@@ -20,3 +20,4 @@ export const hindi = 'हिंदी (hi)';
 export const support_dcs_rename_failed = 'Door43 rename repo failed.  Could not rename repo ‘${old_repo_name}’ to ‘${new_repo_name}’ for user ‘${user_name}’';
 export const support_dcs_create_new_failed = 'Door43 create new failed.  Could not create new repo ‘${new_repo_name}’ for user ‘${user_name}’';
 export const support_dcs_rename_conflict = 'Door43 rename conflict.  Repo ‘${new_repo_name}’ already exists for user ‘${user_name}’';
+export const support_dcs_create_new_conflict = 'Door43 create new repo conflict.  Repo ‘${new_repo_name}’ already exists for user ‘${user_name}’';
```
Comment (mannycolon) [inline src/locale/nonTranslatable.js]: Why is this not translatable? also is the grammar of the text correct?

File: src/js/selectors/index.js [hunk 2]
```diff
@@ -317,6 +317,15 @@ export const getShowProjectInformationScreen = (state) =>
   fromProjectValidationReducer.getShowProjectInformationScreen(
     state.projectValidationReducer);
 
+/**
+ * Checks if show only the project information screen has callback.
+ * @param {Object} state
+ * @return {boolean}
+ */
+export const getShowProjectInformationScreenCallback = (state) =>
```
Comment (mannycolon) [inline src/js/selectors/index.js:325]: What is this? is a callback being stored in a reducer?

File: src/js/reducers/sourceContentUpdatesReducer.js [hunk 3]
```diff
@@ -14,6 +17,11 @@ const SourceContentUpdatesReducer = (state = initialState, action) => {
       ...state,
       resources: [],
     };
+  case consts.SOURCE_CONTENT_UPDATED:
```
Comment (mannycolon) [inline src/js/reducers/sourceContentUpdatesReducer.js]: Should we make the action type more explicit to what it is actually doing? For instance, in this case it is updating the source content update count.

File: src/js/reducers/projectValidationReducer.js [hunk 4]
```diff
@@ -58,6 +59,11 @@ const projectValidationReducer = (state = initialState, action) => {
       ...state,
       onlyShowProjectInformationScreen: action.value,
     };
+  case types.SHOW_ONLY_PROJECT_INFORMATION_SCREEN_CALLBACK:
```
Comment (mannycolon) [inline src/js/reducers/projectValidationReducer.js:62]: Should we find a better name for this? I find it very confusing and cant figure out what it actually means

**Review by mannycolon — CHANGES_REQUESTED** [review]

**Review by PhotoNomad0 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/unfoldingword/translationcore/6537)* status: 0 of 252 files reviewed, 4 unresolved discussions (waiting on @mannycolon and @PhotoNomad0)

___
*[src/locale/nonTranslatable.js, line 23 at r1](https://reviewable.io/reviews/unfoldingword/translationcore/6537#-LtRIt4NX0PaXG1h1AEh-r1-23:-LtVGyQb0OJJFONgvN2L:b-qzwwwq) ([raw file](https://github.com/unfoldingword/translationcore/blob/22cd49acbc16e4d6a3862b84004acd3958964111/src/locale/nonTranslatable.js#L23)):*
<details><summary><i>Previously, mannycolon (Manny Colon) wrote…</i></summary><blockquote>

Why is this not translatable? also is the grammar of the text correct?
</blockquote></details>

These are not strings shown to user, but sent in the submit feedback.  But grammar should be more clear.

**Comment by dsm-git** [comment]
[Linux build 2.1.0 (22cd49a)](https://tc-builds.door43.org/build/release-v2.1.0/translationCore-linux-x64-2.1.0-22cd49a.deb) is ready for download.
[macOS build 2.1.0 (22cd49a)](https://tc-builds.door43.org/build/release-v2.1.0/translationCore-macos-x64-2.1.0-22cd49a.dmg) is ready for download.
[Windows x64 build 2.1.0 (22cd49a)](https://tc-builds.door43.org/build/release-v2.1.0/translationCore-win-x64-2.1.0-22cd49a.setup.exe) is ready for download.
[Windows x32 build 2.1.0 (22cd49a)](https://tc-builds.door43.org/build/release-v2.1.0/translationCore-win-x32-2.1.0-22cd49a.setup.exe) is ready for download.

**Review by PhotoNomad0 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/unfoldingword/translationcore/6537)* status: 0 of 252 files reviewed, 4 unresolved discussions (waiting on @mannycolon and @PhotoNomad0)

___
*[src/js/selectors/index.js, line 325 at r1](https://reviewable.io/reviews/unfoldingword/translationcore/6537#-LtRIt4NX0PaXG1h1AEf-r1-325:-LtVKGdR2nSdVvu_UHxB:b-fzrvjp) ([raw file](https://github.com/unfoldingword/translationcore/blob/22cd49acbc16e4d6a3862b84004acd3958964111/src/js/selectors/index.js#L325)):*
<details><summary><i>Previously, mannycolon (Manny Colon) wrote…</i></summary><blockquote>

What is this? is a callback being stored in a reducer?
</blockquote></details>

Good point - this is not a state we need to act on.  I think I "borrowed" this idea from the submit feedback reducer.

**Review by mannycolon — APPROVED** [review]

**Review by mannycolon — COMMENTED** [review]
:lgtm:

> Reviewed 252 of 252 files at r1, 8 of 8 files at r2.
> *[Reviewable](https://reviewable.io/reviews/unfoldingword/translationcore/6537#-:-LtW20v1CgD1ODJuSZI_:bnfp4nl)* status: all files reviewed, 1 unresolved discussion (waiting on @PhotoNomad0)

___
*[src/locale/nonTranslatable.js, line 23 at r1](https://reviewable.io/reviews/unfoldingword/translationcore/6537#-LtRIt4NX0PaXG1h1AEh-r1-23:-LtVs4GP-ZunA7d8ZqCk:b-pqov82) ([raw file](https://github.com/unfoldingword/translationcore/blob/22cd49acbc16e4d6a3862b84004acd3958964111/src/locale/nonTranslatable.js#L23)):*
<details><summary><i>Previously, PhotoNomad0 (Bruce McLean) wrote…</i></summary><blockquote>

These are not strings shown to user, but sent in the submit feedback.  But grammar should be more clear.
</blockquote></details>

makes sense

**Review by mannycolon — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/unfoldingword/translationcore/6537)* status: :shipit: complete! all files reviewed, all discussions resolved

**Comment by dsm-git** [comment]
[Linux build 2.1.0 (b775824)](https://tc-builds.door43.org/build/release-v2.1.0/translationCore-linux-x64-2.1.0-b775824.deb) is ready for download.
[macOS build 2.1.0 (b775824)](https://tc-builds.door43.org/build/release-v2.1.0/translationCore-macos-x64-2.1.0-b775824.dmg) is ready for download.
[Windows x64 build 2.1.0 (b775824)](https://tc-builds.door43.org/build/release-v2.1.0/translationCore-win-x64-2.1.0-b775824.setup.exe) is ready for download.
[Windows x32 build 2.1.0 (b775824)](https://tc-builds.door43.org/build/release-v2.1.0/translationCore-win-x32-2.1.0-b775824.setup.exe) is ready for download.

**Pull request merged**